OWNERS_ALIASES

```diff
@@ -11,6 +11,7 @@ aliases:
     - woopstar
     - luckysb
     - floryut
+    - pasqualet
```

Comment:
Why global approvers though ? a bit premature I'd say 😄

Reply:
I thought it was mandatory. Fixed!